types/sequencer_set.go

```diff
@@ -134,13 +197,36 @@ func (s *SequencerSet) GetByConsAddress(cons_addr []byte) *Sequencer {
 	return nil
 }
 
+// SequencerListRightOuterJoin returns a set of sequencers that are in B but not in A.
+// CONTRACT: both A and B do not have duplicates!
```

Comment:
not sure I follow this. why is this assumption necessary? worst case diff is empty no?